tests/cartesian_tests/integration_tests/multi_feature_tests/test_code_generation.py

```diff
@@ -586,3 +586,79 @@ def test(out: Field[np.float64], inp: Field[np.float64]):
         backend=backend, aligned_index=(0, 0, 0), shape=(2, 2, 2), dtype=np.float64
     )
     test(out, inp)
+
+
+@pytest.mark.parametrize("backend", ["numpy"])
```

Comment:
Why only numpy?

Reply:
My bad, that was a debug setting, obviously it's all backends